.github/workflows/push.yml

```diff
@@ -357,6 +382,11 @@ jobs:
             echo "::set-output name=BUILD_CHUNKS::$CHUNKS"
           fi
 
+      - name: check pre-release
+        if: contains(github.ref, "pre-release/")
+        run: |
+          echo "AFFECTED_ALL=7913-$GIT_BRANCH" >> $GITHUB_ENV
+
```

Comment:
I'm guessing we don't want this, since we have the other one?

Reply:
It's supposed to be set here and not above. Fixed it